src/components/Container.tsx

```diff
@@ -1,34 +1,49 @@
+import assign = require('lodash/assign');
 import * as React from 'react';
-import styled, { withTheme } from 'styled-components';
+import { withProps } from 'recompose';
+import styled from 'styled-components';
 import {
-	color,
-	fontSize,
-	responsiveStyle,
-	space,
+	maxWidth,
+	MaxWidthProps,
 	textAlign,
+	TextAlignProps,
 } from 'styled-system';
-import { DefaultProps, Theme } from '../common-types';
+import { Theme } from '~/common-types';
```

Comment:
@sradevski Can we use `../` instead? This one fails in the dashboard. There is also in some other places (Img, Pager, etc)

Reply:
@dimitrisnl Yes, this was auto-imported in VSCode, I will update it everywhere, thanks for the info.